CommonConfig.mjs

```diff
@@ -0,0 +1,481 @@
+        "no-useless-escape": ["error"],
+        "no-useless-rename": ["error"],
+        "no-useless-return": ["error"],
+        "no-var": ["error"],
+        "no-void": ["error"],
+        "no-warning-comments": [
+            "warn",
+            {
+                location: "start",
+                terms: [
+                    "todo",
+                    "fixme",
+                ],
+            },
+        ],
+        "no-with": ["error"],
+        "object-shorthand": [
+            "error",
+            "consistent",
+        ],
+        "one-var": [
+            "error",
+            {var: "never", let: "never", const: "never"},
+        ],
+        "operator-assignment": ["error"],
+        "prefer-arrow-callback": ["error"],
+        "prefer-const": [
+            "error",
+            {destructuring: "all", ignoreReadBeforeAssign: false},
+        ],
+        "prefer-destructuring": ["error"],
+        "prefer-exponentiation-operator": ["error"],
+        "prefer-named-capture-group": ["off"],
+        "prefer-numeric-literals": ["error"],
+        "prefer-object-has-own": ["error"],
+        "prefer-object-spread": ["error"],
+        "prefer-promise-reject-errors": ["error"],
+        "prefer-regex-literals": ["error"],
+        "prefer-rest-params": ["error"],
+        "prefer-spread": ["error"],
+        "prefer-template": ["error"],
+        "radix": ["error"],
+        "require-atomic-updates": ["warn"],
+        "require-await": ["warn"],
+        "require-unicode-regexp": ["off"],
+        "require-yield": ["error"],
+        "simple-import-sort/exports": ["warn"],
+        "simple-import-sort/imports": [
+            "warn",
+            {
+                /*
+                // Built-in Node.js modules
+                import fs from "node:fs";
+                import path from "node:path";
+
+                // Framework libraries
+                import {Meteor} from "meteor/meteor";
+                import PropTypes from "prop-types";
+                import React from "react";
+                import ReactDOM from "react-dom";
+
+                // Third-party libraries
+                import axios from "axios";
+                import lodash from "lodash";
+
+                // UI icons
+                import {
+                    faBars,
+                    faFootball,
+                } from "@fortawesome/free-solid-svg-icons";
+                import BarChartIcon from "@mui/icons-material/BarChart";
+                import FoodBankIcon from "@mui/icons-material/FoodBank";
+
+                // Absolute imports (e.g., project-specific aliases)
+                import apiRoutes from "/api/routes";
+                import config from "/config";
+
+                // Relative imports
+                import App from "./components/App";
+                import helperFunction from "./utils";
+
+                // Side-effect imports (e.g., those which have `\u0000` prepended)
```

Comment:
```suggestion
                // Side-effect imports (those which have `\u0000` prepended)
```

Unless not all side-effect imports have `\u0000` prepended?

Reply:
My mistake. I meant to say "i.e.," lol

Reply:
I guess we don't have to add i.e. . will take your suggestion